Makefile

```diff
@@ -8,7 +8,7 @@ build: artifactory/io/projectriff/java/io.projectriff.java
 test:
 	go test -v ./...
 
-acceptance:
+acceptance: acceptance/testdata/builder.toml
```

Comment:
does `aceptance` depend on build? I got this error on a clean checkout:

```
ERROR: reading buildpack.toml from path /Users/trisberg/workspace/projectriff/java-function-buildpack/artifactory/io/projectriff/java/io.projectriff.java/latest/buildpack.toml: open /Users/trisberg/workspace/projectriff/java-function-buildpack/artifactory/io/projectriff/java/io.projectriff.java/latest/buildpack.toml: no such file or directory
make: *** [acceptance] Error 1
```

Reply:
running the acceptance tests is independent of building the builder image. Run `make build` first